### ISSUE TITLE
Only variables should be passed by reference - controllers/Addresses.php

### ISSUE THREAD

**Issue by olimortimer** [gocart/GoCart3#52]
To fix:

```
A PHP Error was encountered

Severity: Notice

Message: Only variables should be passed by reference

Filename: controllers/Addresses.php

Line Number: 73
```

**Comment by gargakk** [comment]
I have this problem too. Temporary solution is to set PHP to 5.6 version.

**Comment by nmormino** [comment]
This issue is fixed with commit 9fd7aed
